add_menufunc.js

```diff
@@ -43,6 +43,327 @@ function MobileMenu() {
+  div16.setAttribute('class', 'move_buttons');
+  div16cont.setAttribute('class', 'move_buttons');
+  button10.setAttribute('class', 'pop_button_edit');
+  button10.setAttribute('type', 'button');
+  button11.setAttribute('class', 'pop_button_edit');
+  button11.setAttribute('type', 'button');
+  button12.setAttribute('class', 'pop_button_edit');
+  button12.setAttribute('type', 'button');
+  divalign16.setAttribute('class', 'align_last_items');
+  div17.setAttribute('class', 'divider');
+  div18.setAttribute('class', 'divider');
+  img21.setAttribute('class', 'portfolio_image');
+  img21.setAttribute('src', './images/portfolio1.svg');
+  img21.setAttribute('alt', 'portfolio_pic');
+  imgcont.setAttribute('src', './images/Counter.svg');
+  imgcont.setAttribute('alt', 'Counter image');
+  imgcont1.setAttribute('src', './images/Counter.svg');
+  imgcont1.setAttribute('alt', 'Counter image');
+  p21.setAttribute('class', 'pop_content');
+  div19.setAttribute('class', 'buttons_live_pop');
+  button13.setAttribute('class', 'padding_live live_link1');
+  button13.setAttribute('onclick', "window.location.href='https://medini-rajendra.github.io/project3_microverse';");
+  div20.setAttribute('class', 'divider_live');
+  button14.setAttribute('class', 'padding_live live_link2');
+  button14.setAttribute('onclick', "window.location.href='https://github.com/Medini-Rajendra/Mobile_Webpage_Menu';");
+  divcover16.setAttribute('class', 'aligndiv16elems');
+
+  h34.textContent = 'Tonic';
+  h44cont.textContent = 'CANOPY';
+  div15.textContent = '×';
+  licont1.innerHTML = '\u00A0\u00A0Back End Dev';
+  licont3.innerHTML = '\u00A0\u00A02015';
+  button10.textContent = 'html';
+  button11.textContent = 'css';
+  button12.textContent = 'javascript';
+  p21.textContent = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent';
+  button13.textContent = 'See live';
+  button14.textContent = 'See source';
+}
+
+function PopProjectWin(div2c, image, title, subtit1, subtit2, subtit3, iml, tech0, tech1, tech2) {
+  this.divclass = div2c;
+  this.title = title;
+  this.image = image;
+  this.subtit1 = subtit1;
+  this.subtit2 = subtit2;
+  this.subtit3 = subtit3;
+  this.imglinkclass = iml;
+  this.tech0 = tech0;
+  this.tech1 = tech1;
+  this.tech2 = tech2;
+}
+
+const project0 = new PopProjectWin(
+  'grid-item item1',
+  './images/portfolio1.svg',
+  'Tonic',
+  'CANOPY',
+  'Back End Dev ',
+  '2015',
+  'A daily selection of privately <br> personlized reads; no accounts or <br> sign-ups required',
+  'html',
+  'css',
+  'javascript',
+);
+
+const project1 = new PopProjectWin(
+  'grid-item item2',
+  './images/portfolio2.svg',
```

Comment:
- [ ] **REQUIRED:** The popup doesn't work in mobile mode, and images move outside of the layout. It will be nice to fix it.
![image](https://user-images.githubusercontent.com/2903437/137417461-97b7b41d-254c-4b28-a59c-603bb36ebdf6.png)

Reply:
This has been fixed. The popup works and the image layout has been fixed.